src/FixedDataTableRow.js

```diff
@@ -516,96 +553,4 @@ class FixedDataTableRowImpl extends React.Component {
   };
 }
 
-class FixedDataTableRow extends React.Component {
-  static propTypes = {
-    isScrolling: PropTypes.bool,
-
-    /**
-     * Height of the row.
-     */
-    height: PropTypes.number.isRequired,
-
-    /**
-     * Z-index on which the row will be displayed. Used e.g. for keeping
-     * header and footer in front of other rows.
-     */
-    zIndex: PropTypes.number,
-
-    /**
-     * The vertical position where the row should render itself
-     */
-    offsetTop: PropTypes.number.isRequired,
-
-    /**
-     * Pass false to hide the row via CSS
-     */
-    visible: PropTypes.bool.isRequired,
-
-    /**
-     * Width of the row.
-     */
-    width: PropTypes.number.isRequired,
-  };
-
-  constructor(props) {
-    super(props);
-    this._initialRender = true;
-  }
-
-  componentDidMount() {
-    this._initialRender = false;
-  }
-
-  shouldComponentUpdate(nextProps) {
-    // only skip updates while scrolling
-    if (!nextProps.isScrolling) {
-      return true;
-    }
-
-    // if row's visibility has changed, then update it
-    if (this.props.visible !== nextProps.visible) {
-      return true;
-    }
-
-    // if row is still not visible then no need to update
-    if (!nextProps.visible) {
-      return false;
-    }
-
-    // if offsets haven't changed for the same row while scrolling, then skip update
-    return !(
-      nextProps.isScrolling &&
-      this.props.index === nextProps.index &&
-      this.props.offsetTop === nextProps.offsetTop &&
-      this.props.scrollLeft === nextProps.scrollLeft
-    );
-  }
-
-  render() /*object*/ {
-    const { offsetTop, zIndex, ...rowProps } = this.props;
-
-    var style = {
-      width: this.props.width,
-      height: this.props.height,
-      zIndex: zIndex ? zIndex : 0,
-    };
-    if (!rowProps.visible) {
-      style.display = 'none';
-    }
-    FixedDataTableTranslateDOMPosition(
-      style,
-      0,
-      offsetTop || 0,
-      this._initialRender,
-      this.props.isRTL
-    );
-
-    return (
-      <div style={style} className={cx('fixedDataTableRowLayout/rowWrapper')}>
-        <FixedDataTableRowImpl {...rowProps} />
-      </div>
-    );
-  }
-}
-
 export default FixedDataTableRow;
```

Comment:
Now there is no need of this wrapper div .We have provided the styling and the className to the corresponding div .So we can remove this FixedDataTableRow class component and rename FixedDataTableRowImpl to FixedDataTableRow as we are passing all the props from FixedDataTable to FixedDataTableRow.